### ISSUE TITLE
Email template preview & sending error when a table field is present in the form

### ISSUE THREAD

**Issue by ljclark** [verbb/formie#882]
When trying to preview an email notification when the form contains a table field, I'm getting the following error:

```
Error: Request failed with status code 500
Unexpected token "name" of value "if" ("end of statement block" expected).
```

Seems to be connected to line 35 in here: `vendor/verbb/formie/src/templates/_special/email-template/fields/table.html` - removing the `if option.value == cellValue` condition from the forloop seems to resolve it, allowing email previews and the email to the send successfully.

### Steps to reproduce

1. Create a multipage form
2. Add in a table field to your form
3. Include the table field in your email notification
4. Try to preview or send the email notification. Previewing will show an error, sending will create a failed task in the queue, 

### Form settings
Please pick from the options below.

Multi-page form: Yes
Submission Method: Ajax
Client-side Validation: Yes
Custom Form Templates: No

### Additional info

Plugin version: dev-craft-4 as 2.0.0-beta.6 (also happens on 2.0.0-beta.6)
Craft version: Craft Pro 4.0.2
Multi-site: No

**Comment by engram-design** [comment]
Ah yes, Craft 4 has removed the for-if, I missed this one!

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by ljclark** [comment]
10 minute patch 🚀 🔥   that's sorted it, thankyou!

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.7](https://github.com/verbb/formie/releases/tag/2.0.0-beta.7)
